decidim-blogs/spec/system/explore_posts_spec.rb

```diff
@@ -54,12 +54,41 @@
 
   describe "show" do
     let(:posts_count) { 1 }
-    let!(:post) { create(:post, component: component) }
+    let(:author) { organization }
+    let!(:post) { create(:post, component: component, author: author) }
 
     before do
       visit resource_locator(post).path
     end
 
+    context "when author is an organization" do
+      it "shows 'Official' as the author" do
```

Comment:
What I was expecting here is, being consistent with other components, like "Official meeting", "Official proposal" and "Official debate". On this case it should be "Official blogspot" or something like that.

Reply:
Done, thanks!